build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.2'
+        classpath 'com.android.tools.build:gradle:3.5.1'
```

Comment:
revert this change

Reply:
The master branch has the same version 3.5.1, Please check this